dipy/workflows/tracking.py

```diff
@@ -134,6 +139,8 @@ def run(self, pam_files, stopping_files, seeding_files,
             Path to images (e.g. FA) used for stopping criteria for tracking.
         seeding_files : string
             A binary image showing where we need to seed for tracking.
+        use_binary_mask : bool, optional
+            If True, uses a binary tissue classifier.
```

Comment:
Maybe add here:
"If the provided `stopping_files` are not binary, the `stopping_thr` will be used to binarize the images.

Reply:
Thanks for the review @gabknight. Done in 32ba095.